apps/src/maze/StepButton.jsx

```diff
@@ -0,0 +1,20 @@
+var msg = require('./locale');
+
+var StepButton = function (props) {
+  var classes = 'launch float-right';
+  if (!props.showStepButton) {
+    classes += ' hide';
+  }
+  return (
+    <button id="stepButton" className={classes}>
```

Comment:
It will be protected by parent (GameButtons) being protected right now. Sticking this in a div actually changes the layout. I wonder if we should have made (or should still make) a ProtectedStatefulSpan